templates/16_update.tpl

```diff
@@ -48,7 +48,15 @@ func (o *{{$tableNameSingular}}) Update(exec boil.Executor, whitelist ... string
 	{{$varNameSingular}}UpdateCacheMut.RUnlock()
 
 	if !cached {
-		wl := strmangle.UpdateColumnSet({{$varNameSingular}}Columns, {{$varNameSingular}}PrimaryKeyColumns, whitelist)
+		wl := strmangle.UpdateColumnSet(
+			{{if ne .DriverName "mssql" -}}
+			{{$varNameSingular}}Columns,
+			{{- else -}}
+			{{$varNameSingular}}ColumnsWithoutAuto,
```

Comment:
Another place that would be cleaned up by coercing "autogenerated" into default/primary key fields as the other dbs if possible.